### ISSUE TITLE
Customized scoped package ranking?

### ISSUE THREAD

**Issue by daltonamitchell** [benmosher/eslint-plugin-import#949]
Is there a way to differentiate scoped packages from other 'external' packages? For example, I'd like to enforce scoped imports this way:

```js
// This is ok
import React from 'react'
import @org/package

...
// This should fail
import @org/package
import React from 'react'
```

Our company has internal scoped packages and we'd like to enforce the ordering of their imports as well. I'm happy to do the work if this functionality doesn't currently exist. I just wanted to make sure I can't already do this and you're open to adding a feature like this.

**Comment by benmosher** [comment]
Sounds like a reasonable enhancement; however, most `import/order` issues/PRs are currently stuck as they seem to generate a lot of discussion and draw a lot of perspectives, and I don't personally use it so I don't have much to contribute. 😅

I probably won't merge any PRs for `import/order` anytime soon, I'm considering spinning it back out into its own project to be maintained separately. (it was originally @jfmengels' independent plugin)

**Comment by benmosher** [comment]
FWIW it’s looking like we probably won’t spin it out after all, so you can feel free to reopen 👍🏻

**Comment by daltonamitchell** [comment]
Would be curious if others need this or some form of flexible config options
